typespec/Applications.Core/extenders.tsp

```diff
@@ -37,7 +37,16 @@ using OpenAPI;
 namespace Applications.Core;
 
 @doc("Response for list secrets API call")
-model ExtenderListSecretResponse {}
+model ExtenderListSecretResponse is ExtenderSecret;
```

Comment:
Added this to test if the **listSecrets** would be created. And it did get created after we remove the empty model.